src/cloudinary-selector.js

```diff
@@ -123,18 +97,34 @@ function initCustomElement() {
         {
           cloud_name: config.cloudName,
           api_key: config.apiKey,
-          default_transformations: config.defaultTransformations,
-          button_class: "btn btn--primary",
-          button_caption: "Select Assets",
```

Comment:
use the previous configuration, see next comment for details.

`default_transformations: config.defaultTransformations`